content/docs/5.1/reference/faq.md

```diff
@@ -9,141 +9,143 @@ menu:
 
 ## What is TiKV?
 
-TiKV is a distributed Key-Value database that features in geo-replication, horizontal scalability, consistent distributed transactions and coprocessor support.
+TiKV is a distributed Key-value database that features in geo-replication, horizontal scalability, consistent distributed transactions, and coprocessor support.
 
 ## How do I run TiKV?
 
-For further information, see [TiKV in 5 Minutes](../../concepts/tikv-in-5-minutes/) for deploying a TiKV testing cluster, or [Install TiKV](../../deploy/install/install/) for deploying TiKV in production.
+If you would like to deploy a TiKV cluster for test environments, you can refer to [TiKV in 5 Minutes](../../concepts/tikv-in-5-minutes/). Besides, for production environments, you can follow the instructions in [Install TiKV](../../deploy/install/install/).
 
 ## When to use TiKV?
 
-TiKV is at your service if your applications require:
+TiKV is the service you need if your applications require the following features:
```

Comment:
I revised it because "at your service" might sound a little bit young and interesting, while other parts are so official 😂 But I think both can deliver the same meaning. How do you think? @shichun-0415

Reply:
The original is more vivid and catchy. I think we can keep it.